src/bootstrap/src/core/build_steps/dist.rs

```diff
@@ -476,7 +476,7 @@ impl Step for Rustc {
             let man_src = builder.src.join("src/doc/man");
             let man_dst = image.join("share/man/man1");
 
-            // don't use our `bootstrap::{copy, cp_r}`, because those try
+            // don't use our `bootstrap::{cp_r}`, because those try
```

Comment:
Should this be `copy_internal` instead? (assuming that's the new name of `copy`)

Reply:
uh, probably